packages/pg/test/integration/connection/test-helper.js

```diff
@@ -6,7 +6,11 @@ var utils = require(__dirname + '/../../../lib/utils')
 var connect = function (callback) {
   var username = helper.args.user
   var database = helper.args.database
-  var con = new Connection({ stream: new net.Stream() })
+  var client_encoding = helper.args.client_encoding
```

Comment:
I see you're passing this in here which is cool, but were you intending on setting this in CI to test both permutations?  AFAIK this will always still default to `utf-8` for now?

Reply:
I haven't considered how to test it. Would it be better to write a few targeted integration and unit tests instead of running all the tests with different client encodings?

Reply:
> Would it be better to write a few targeted integration and unit tests instead of running all the tests with different client encodings?

yeah i think so - running the whole test suite again just to test a different encoding is overkill IMO